dev-tools/packaging/templates/docker/Dockerfile.elastic-agent.tmpl

```diff
@@ -17,6 +17,14 @@ RUN for iter in {1..10}; do \
 
 COPY beat {{ $beatHome }}
 
+{{- if (and (contains .buildFrom "redhat/ubi") (contains .buildFrom "-minimal")) }}
+RUN for iter in {1..10}; do \
+        microdnf update -y && \
+        microdnf install -y findutils && \
+        microdnf clean all && \
+        exit_code=0 && break || exit_code=$? && echo "microdnf error: retry $iter in 10s" && sleep 10; done; (exit $exit_code)
+{{- end }}
```

Comment:
Sorry for not spotting this earlier, but a couple command below there is this block

```
{{- if (and (contains .from "redhat/ubi") (contains .from "-minimal")) }}
RUN for iter in {1..10}; do \
        microdnf update -y && \
        microdnf install -y tar gzip findutils shadow-utils ca-certificates gawk libcap xz && \
        microdnf clean all && \
        exit_code=0 && break || exit_code=$? && echo "microdnf error: retry $iter in 10s" && sleep 10; done; (exit $exit_code)
{{- end }}
```

maybe we can bring it upper and merge it with this one and thus save one layer from the final image, wdyt? 🙂

Reply:
oops yes, my mistake, I said the final image which is incorrect. I meant we save one layer in general that is redundant as it installs the same things thus we could bring just the other one higher and have the required `find` sooner

Reply:
but still this would not be available during stage 2 because base image of stage 2 is not the result of a state one, it's more of a parallel images

Reply:
now I see what you mean there is FROM between these... ignore this comment and thanks for "showing" me